### ISSUE TITLE
Cannot install library on Python 3.10

### ISSUE THREAD

**Issue by yuvadm** [yonatanp/electrasmart#12]
Attempting to install the library in a virtualenv running Python 3.10 fails with the following trace, which seems to come from `quicklib`:

```
  Complete output (11 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-6ih54_ro/electrasmart_2ba398d1a3774079a409a3ece0ad7f9d/setup.py", line 10, in <module>
      _sys.path.insert(0, zips[0]); import quicklib; _sys.path.pop(0)
    File "quicklib_incorporated.v0.4.4.zip/quicklib/__init__.py", line 1, in <module>
    File "quicklib_incorporated.v0.4.4.zip/quicklib/versioning.py", line 2, in <module>
    File "quicklib_incorporated.v0.4.4.zip/past/builtins/__init__.py", line 43, in <module>
    File "quicklib_incorporated.v0.4.4.zip/past/builtins/noniterators.py", line 24, in <module>
    File "quicklib_incorporated.v0.4.4.zip/past/types/__init__.py", line 25, in <module>
    File "quicklib_incorporated.v0.4.4.zip/past/types/oldstr.py", line 5, in <module>
  ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
```

**Comment by yuvadm** [comment]
In the meantime I wrote a minimal `setup.py` so that I can run a custom install, forked at https://github.com/yuvadm/electrasmart

**Comment by yonatanp** [comment]
@yuvadm fine work there!
If you can confirm that everything is working with the new setup.py, including the cli scripts, please send a PR and I will merge it and release a new version next week. We don’t seem to be using much of quicklib’s capabilities so we might as well drop the dependency.

**Comment by yuvadm** [comment]
Sounds good! Yeah it works like a charm for me, will make a PR out of my fork

**Comment by yonatanp** [comment]
Released version 0.7 including this fix.
Released `electrasmart-custom-component` version 2022.4.25 to use it.
